DECISIONS.md

```diff
@@ -0,0 +1,40 @@
+# Package decisions
+
+## Decision
+Regarding the ideation below, we ended up going with ...
+
+## Design
+
+### Goals
+1. User accessibility for the use case of integrating the graphing method in computational research.
+    - This goal means ease of ease of use and ease of installation into their environment.
+    - **We agree this is the main goal of the project.**
+1. Maintainability
+    - It is easy to make updates to the code, hopefully from its main `npm` package.
+    - One realism we already foresee is that the `npm` package exists to be extensible for use cases beyond the current one--the current use case requires some tailoring to intercept the graph `svg` and save it to a file, potentially meaning we store a version of the package in this repo to make adjustments for serverless graph creation.
+1. Performance
+    - The graphing method runs headlessly and should be able to handle large data sets as well as be integrated into larger analyses, pipelines, parallel processing, etc.
```

Comment:
I guess one question is how large is large? Maybe we could run some benchmarks on twitter data.

Reply:
If we are rendering the html in some hidden browser I have a feeling that will be the bottle neck

Reply:
I encourage creating an issue if there's definitely a 'big data' use case we want to benchmark for.